### PR TITLE
UI/Service-map-GUI-improvements

### PR DESCRIPTION
**Pull request by leon-up9**
servicemap new design

### CODE REVIEW

File: ui-common/src/variables.module.scss [hunk 1]
```diff
@@ -7,6 +7,10 @@ $blue-color: #205CF5;
 $light-blue-color: #BCCEFD;
 $success-color: #27AE60;
 $failure-color: #EB5757;
+
```
Comment (leon-up9) [inline ui-common/src/variables.module.scss:10]: added from Ent

**Comment by AmitUp9** [comment]
features waiting to develop: 
- edges color 
- filter button style

File: ui-common/src/components/ServiceMapModal/ServiceMapModal.module.sass [hunk 2]
```diff
@@ -1,64 +1,111 @@
...
+@import "../../mixins"
+
+.closeIcon
+  position: absolute
+  right: 20px
+  top: 20px
 
 .modalContainer
   display: flex
+  width: calc(100% - 15 - 15px)
```
Comment (lirazyehezkel) [inline ui-common/src/components/ServiceMapModal/ServiceMapModal.module.sass:11]: -15 -15px ?

File: ui-common/src/components/ServiceMapModal/ServiceMapModal.module.sass [hunk 3]
```diff
@@ -1,64 +1,111 @@
...
   display: flex
   flex-direction: column
   margin-right: 10px
   width: 100%
+  border-radius: 4px
 
 .servicesFilterSearch
-  width: calc(100% - 10px)
-  max-width: 300px
+  width: -moz-available
```
Comment (lirazyehezkel) [inline ui-common/src/components/ServiceMapModal/ServiceMapModal.module.sass:76]: why do you need this?

File: ui-common/src/mixins.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+@mixin sub-section-header{
```
Comment (lirazyehezkel) [inline ui-common/src/mixins.scss:1]: check if @mixin is neccessery here

File: ui-common/src/components/ServiceMapModal/ServiceMapModal.module.sass (hunk 3 above)
Comment (leon-up9) [reply ui-common/src/components/ServiceMapModal/ServiceMapModal.module.sass:76]: supporting mozilla browser

**Review by lirazyehezkel — APPROVED** [review]

**Pull request merged**